### DISCUSSION TITLE
OOO

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#763] [General]
Miami F1 Grand Prix is in town this weekend and I'll be spending most time either at the track or related events - don't expect much updates till next week :)

**Comment by brknsoul** [comment]
Hope you have a fun time! A well deserved break!

**Comment by TKS4** [comment]
Can't wait, hopefully it'll be a bit more exciting than just Red Bull moping everything up as always >_<

**Comment by iDeNoh** [comment]
Everything is in a pretty excellent state at the moment, enjoy yourself my guy, you deserve it!

**Comment by nekoworkshop** [comment]
Hi would you mind adding your days off to the office OOO calendar so we can plan our sprint accordingly😳? /s

Just kidding. Enjoy the race Vlad! 🏁🏁🏁

**Comment by Tillerz** [comment]
Now that is fun! Go enjoy it, woohoo! \o/

**Comment by Centurion-Rome** [comment]
I hope you will get some fun. You need some break also ;)
Enjoy it!

**Comment by Aptronymist** [comment]
You've earned it, relax and enjoy!
